packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -44,7 +44,7 @@ import { ChartAxisDirection } from '../chartAxisDirection';
 import { CartesianCrossLine } from '../crossline/cartesianCrossLine';
 import type { CrossLine } from '../crossline/crossLine';
 import type { AnimationManager } from '../interaction/animationManager';
-import type { InteractionEvent } from '../interaction/interactionManager';
+import type { PointerInteractionEvent as InteractionEvent } from '../interaction/interactionManager';
```

Comment:
Nit: Might be better to import as the actual name of the interface, to save confusion later?

Reply:
Agreed. I've got a follow up PR for this nit https://github.com/ag-grid/ag-charts/pull/1361/files. This was just a temporary workaround.